src/mappings/entities/mapClaim.ts

```diff
@@ -18,7 +15,7 @@ enum ClaimScopeTypeEnum {
 }
 
 type ClaimParams = {
-  claimExpiry: bigint;
+  claimExpiry: bigint | undefined;
```

Comment:
I think this means the same thing, but if not OK to leave.
```suggestion
  claimExpiry?: bigint;
```